lib/src/main/java/io/ably/lib/http/Http.java

```diff
@@ -37,769 +37,717 @@
-     * @param url
-     * @param proxy
-     * @param method
-     * @param headers
-     * @param requestBody
-     * @param responseHandler
-     * @return
-     * @throws AblyException
-     */
-    <T> T httpExecuteWithRetry(URL url, String method, Param[] headers, RequestBody requestBody, ResponseHandler<T> responseHandler, boolean allowAblyAuth) throws AblyException {
-        boolean authPending = true, renewPending = true, proxyAuthPending = true;
-        while (true) {
-            try {
-                return httpExecute(url, getProxy(url), method, headers, requestBody, true, responseHandler);
-            } catch (AuthRequiredException are) {
-                if (are.authChallenge != null && allowAblyAuth) {
-                    if (authPending) {
-                        authorise(false);
-                        authPending = false;
-                        continue;
-                    }
-                    if (are.expired && renewPending) {
-                        authorise(true);
-                        renewPending = false;
-                        continue;
-                    }
-                }
-                if (are.proxyAuthChallenge != null && proxyAuthPending && proxyAuth != null) {
-                    proxyAuth.processAuthenticateHeaders(are.proxyAuthChallenge);
-                    proxyAuthPending = false;
-                    continue;
-                }
-                throw are;
-            }
-        }
-    }
-
-    /**
-     * Handle HTTP response
-     * @param conn
-     * @param credentialsIncluded
-     * @param response
-     * @param responseHandler
-     * @return
-     * @throws AblyException
-     */
-    private <T> T handleResponse(HttpURLConnection conn, boolean credentialsIncluded, Response response, ResponseHandler<T> responseHandler) throws AblyException {
-        if (response.statusCode == 0) {
-            return null;
-        }
-
-        if (response.statusCode >= 500 && response.statusCode <= 504) {
-            throw AblyException.fromErrorInfo(ErrorInfo.fromResponseStatus(response.statusLine, response.statusCode));
-        }
-
-        if (response.statusCode < 200 || response.statusCode >= 300) {
-            /* get any in-body error details */
-            ErrorInfo error = null;
-            if (response.body != null && response.body.length > 0) {
-                String bodyText = new String(response.body);
-                try {
-                    ErrorResponse errorResponse = ErrorResponse.fromJSON(bodyText);
-                    if (errorResponse != null) {
-                        error = errorResponse.error;
-                    }
-                } catch (JsonParseException jse) {
-                    /* error pages aren't necessarily going to satisfy our Accept criteria ... */
-                    System.err.println("Error message in unexpected format: " + bodyText);
-                }
-            }
+			if(requestBody != null) {
```

Comment:
This looks like a repeat of what happened before, you're committing white space changes for no reason?